tasks/usemin.js

```diff
@@ -122,7 +122,8 @@ module.exports = function (grunt) {
 
     // var locator = options.revmap ? grunt.file.readJSON(options.revmap) : function (p) { return grunt.file.expand({filter: 'isFile'}, p); };
     var locator = getLocator(grunt, options);
-    var revvedfinder = new RevvedFinder(locator);
+    var baseURL = options.baseURL || '';
+    var revvedfinder = new RevvedFinder(locator, baseURL);
```

Comment:
`var revvedfinder = new RevvedFinder(locator, options.baseURL);`

If options.baseUrl is not defined it will pass in a undefined (falsey value)